python/foglamp/services/core/api/filters.py

```diff
@@ -516,6 +405,7 @@ def new_items(lst1, lst2):
     else:
         return web.json_response({'result': "Filter {} deleted successfully".format(filter_name)})
 
+
 async def delete_filter_pipeline(request: web.Request) -> web.Response:
     """ DELETE filter pipeline
```

Comment:
We should not make http request here, its a call within same API. so better we clone to PUT empty or create a common def to be called for PUT and `DELETE  == PUT empty`

Reply:
request object cloning does allow adding a PUT/POST payload, hence this approach. See https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.BaseRequest.clone.